ftw/news/browser/resources/news.scss

```diff
@@ -2,69 +2,89 @@
 @include portal-type-font-awesome-icon(ftw-news-newsfolder, newspaper-o);
 @include portal-type-font-awesome-icon(ftw-news-newslistingblock, newspaper-o);
 
-.portletItemDate {
-  color: $color-text-light;
-  font-size: $font-size-small;
-  margin-bottom: $margin-heading-vertical;
-}
+$leadimage-width: 200px;
```

Comment:
@bierik Yes, it is possible by registering a dynamic scss resouce which declares the variable, similar to
- https://github.com/4teamwork/ftw.theming/blob/master/ftw/theming/variables.py#L9
- https://github.com/4teamwork/ftw.theming/blob/master/ftw/theming/resources.zcml#L6
---
Which imaging scale is used for news?